src/components/modal/modal.tsx

```diff
@@ -26,6 +27,11 @@ export default function Modal({
     >
       <Button onClick={onModalClose} className="close modal-close" />
       <div>
+        {title === ModalTitle.Congratulations && (
+          <div className="modal-icon">
```

Comment:
It looks invalid because if you have a "regret modal" with some edge case, you'll add another condition for the title as well. Let's pass it via props because the modal as a component should know nothing about some particular modal title.

Reply:
Changed the component to pass the icon as a prop.